File: src/main/java/sage/web/auth/RequireLoginException.java
package sage.web.auth;

public class RequireLoginException extends RuntimeException {
  private static final long serialVersionUID = -7612510838405643774L;

  public RequireLoginException() {
    super();
  }

  public RequireLoginException(String message, Throwable cause, boolean enableSuppression,
      boolean writableStackTrace) {
    super(message, cause, enableSuppression, writableStackTrace);
  }

  public RequireLoginException(String message, Throwable cause) {
    super(message, cause);
  }

  public RequireLoginException(String message) {
    super(message);
  }

  public RequireLoginException(Throwable cause) {
    super(cause);
  }

}

File: src/main/kotlin/sage/web/auth/RequireLoginException.kt
package sage.web.auth

class RequireLoginException : RuntimeException {

  constructor() : super() {
  }

  constructor(message: String, cause: Throwable, enableSuppression: Boolean,
              writableStackTrace: Boolean) : super(message, cause, enableSuppression, writableStackTrace) {
  }

  constructor(message: String, cause: Throwable) : super(message, cause) {
  }

  constructor(message: String) : super(message) {
  }

  constructor(cause: Throwable) : super(cause) {
  }

  companion object {
    private val serialVersionUID = -7612510838405643774L
  }

}
